core/modules/user/user.install

```diff
@@ -1122,12 +1122,24 @@ function user_update_1011() {
   $config->save();
 }
 
+
 /**
- * Update 'Find user accounts' to 'Manage user accounts'
+ * Update 'People' menus to 'User accounts'.
  */
 function user_update_1012() {
   $config = config('views.view.user_admin');
-	if ($config->get('display.default.display_options.title') == 'User accounts') {
+  if ($config->get('display.page.display_options.path') == 'admin/people/list') {
+    $config->set('display.page.display_options.path', 'admin/accounts/list');
+    $config->save();
+  }
+}
```

Comment:
This change needs to be done as update_1015(), existing sites would have already run update_1012() and update_1013() at this point.

Reply:
Hm, not sure why I said update_1015(), looks like the next number is update_1014()?

Reply:
fixed.